apps/condo/domains/ticket/schema/TicketHint.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Generated by `createschema ticket.TicketHint 'organization:Relationship:Organization:CASCADE; name?:Text; properties:Relationship:Property:SET_NULL; content:Text;'`
+ */
+
+const { Text, Relationship } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+const { dvAndSender } = require('@condo/domains/common/schema/plugins/dvAndSender')
+const access = require('@condo/domains/ticket/access/TicketHint')
+const { ORGANIZATION_OWNED_FIELD } = require('@condo/domains/organization/schema/fields')
+const xss = require('xss')
+
+const TicketHint = new GQLListSchema('TicketHint', {
+    schemaDoc: 'Help information related to a property or group of properties',
```

Comment:
Could you bring some examples of what can be presented in information text?

```suggestion
    schemaDoc: 'Textual information in free format related to a property or group of properties, for example…',
```